modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/portlet/shared/task/SearchHttpUtil.java

```diff
@@ -52,6 +52,9 @@ public static String getCompleteOriginalURL(
 		else {
 			requestURL = String.valueOf(httpServletRequest.getRequestURL());
 
+			requestURL = _portal.getAbsoluteURL(
```

Comment:
do you want to SF this so that we only assign requestURL once?

Reply:
Sure

Reply:
@BryanEngler - force pushed